vscode/README-DEV.md

```diff
@@ -5,3 +5,5 @@ of upcoming features.
 
 To install the tested version of this extension, visit <https://marketplace.visualstudio.com/items?itemName=quantum.qsharp-lang-vscode>.
 The README for the stable build contains details on features and usage.
+
+For more information about the Modern QDK and Azure Quantum, visit [https://aka.ms/AQ/Documentation](https://aka.ms/AQ/Documentation).
```

Comment:
I'd leave this readme minimal and referring to the stable extension for more info (as it does already).